packages/ui/primitives/document-flow/add-signers.tsx

```diff
@@ -38,9 +41,14 @@ export const AddSignersFormPartial = ({
   recipients,
   fields: _fields,
   onSubmit,
+  creatorEmail,
+  creatorName,
+  selfSign,
```

Comment:
As mentioned we can use `useSession` here too

Reply:
How can I use that? Because it is a package & it do not have next js.

If there is a way please guide.

Reply:
You can just pass the `user` down as a prop since the parent appears to already appears to use it